pennylane_lightning/core/_version.py

```diff
@@ -16,4 +16,4 @@
 Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.42.0-dev3"
+__version__ = "0.42.0-dev4"
```

Comment:
Do we also have a PR for this to the postfix branch? Or are we cherry picking the additions?
@maliasadi

Reply:
CC @maliasadi

Reply:
We cherry picked this in PR #1137